toptal/picasso
# fix(Autocomplete): add inputProps type to the definition of Autocomplete

### Description

the definition of inputProps was removed from the Autocomplete props definition.

### How to test

as `inputProps` prop is used in Autocomplete (see: https://github.com/toptal/picasso/blob/e798f93b4c8f8764c49adacd15bd4addca2ce1ed/packages/picasso/src/Autocomplete/Autocomplete.tsx#L273)

we also have to add information about `inputProps` prop to the Autocomplete props definition.

### Screenshots

before:

<img width="819" alt="screenshot_input-props_definition" src="https://user-images.githubusercontent.com/76203853/124737794-fb1f1d80-df20-11eb-84e9-4509243ce8db.png">



### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
